contracts/validator-manager/ValidatorMessages.sol

```diff
@@ -202,19 +224,34 @@ library ValidatorMessages {
         returns (bytes32, bytes memory)
     {
         if (validationPeriod.blsPublicKey.length != 48) {
-            revert InvalidMessageLength();
+            revert InvalidBLSPublicKey();
         }
 
         // solhint-disable-next-line func-named-parameters
         bytes memory res = abi.encodePacked(
             CODEC_ID,
             REGISTER_SUBNET_VALIDATOR_MESSAGE_TYPE_ID,
             validationPeriod.subnetID,
+            uint32(validationPeriod.nodeID.length),
```

Comment:
Nothing we can do here but the fact that now we have messages with multiple dynamic fields means we can't use the current version of generator in #482 here any longer 😢

Reply:
This is actually false, we can still do it but it's more involved